sherlock/resources/data.json

```diff
@@ -1851,14 +1786,6 @@
     "username_claimed": "blue",
     "username_unclaimed": "noonewouldeverusethis"
   },
-  "babyblogRU": {
```

Comment:
BabyBlogRU was a real headache for me, maybe because I was out of energy by that time.